doc/user/content/integrations/postgres-bastion.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "SSH Bastion (PostgreSQL)"
+description: "How to connect Postgres to Materialize through an SSH Bastion connection"
+menu:
+  main:
+    parent: "integration-guides"
+    name: "SSH Bastion (PostgreSQL)"
+---
+
+Materialize can connect to a Postgres database through a secure SSH bastion server. Through this guide, you will learn how to:
+* Create the connections
+* Create the source
+* Configure Materialize public keys on the bastion server
+
+### Pre-requirements
+
+Before following the steps, check meeting the next items:
+
+* Running Postgres database
+* SSH bastion server
+    * It must be accessible from Materialize and have access to Postgres
+* Materialize's region enabled
+
+### Steps
+
+1. Create in Materialize the [SSH connection](/sql/create-connection/#example-4) to the bastion server:
+    ```sql
+    CREATE CONNECTION ssh_connection TO SSH TUNNEL (
+        HOST '<SSH_BASTION_HOST>',
+        USER '<SSH_BASTION_USER>',
+        PORT 1
```

Comment:
Maybe we should also mention this for the port: `<SSH_BASTION_PORT>` rather than jus `1`. What do you think?